pypowsybl/network/impl/network.py

```diff
@@ -390,9 +390,19 @@ def write_network_area_diagram(self, svg_file: PathOrStr, voltage_level_ids: Uni
         _pp.write_network_area_diagram_svg(self._handle, svg_file, '' if metadata_file is None else path_to_str(metadata_file),
                                            voltage_level_ids, depth, high_nominal_voltage_bound, low_nominal_voltage_bound, nad_p)
 
+    def _create_nad_positions_c_dataframe(self, df: DataFrame) -> _pp.Dataframe:
+        nad_positions_metadata=[_pp.SeriesMetadata('id',0,True,False,False),
+                  _pp.SeriesMetadata('x',1,False,False,False),
+                  _pp.SeriesMetadata('y',1,False,False,False),
+                  _pp.SeriesMetadata('shiftX',1,False,False,False),
+                  _pp.SeriesMetadata('shiftY',1,False,False,False),
+                  _pp.SeriesMetadata('connectionShiftX',1,False,False,False),
+                  _pp.SeriesMetadata('connectionShiftY',1,False,False,False)]
```

Comment:
the parameters about the legend should be more explicit, something like legendShiftX, legendConnectionShiftX?